### DISCUSSION TITLE
Add the ability to search for hotkeys in the settings

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#4950] [Ideas] [answered]
There are now about 70 hotkeys in the settings list. Instead of scrolling through the list, it would be convenient to enter what you are interested in and see matches from the list of hotkeys in the search.

For example, if I enter "zoom", then the search results should include these lines (in addition to other matches):
![Screenshot 2023-11-06 at 14 47 21](https://github.com/Chatterino/chatterino2/assets/60105060/dbeff939-cbb6-4949-b5ff-d68ade82c865)

What do you think?

**Comment by Mm2PL** [comment]
Also searching by keys would be nice. We could probably take some inspiration from software like Intellij on how to make this work UI/UX wise:
![image](https://github.com/Chatterino/chatterino2/assets/25011746/f5778750-2c56-45ef-aeeb-0092f609bed0)

**Comment by 8thony** [comment, marked as answer]
- Searching hotkey name via global search and searching for a key via a special field on the top right of the Hotkeys page 
- Implemented since #5884
